src/lib/storage/chunk.cpp

```diff
@@ -217,9 +217,9 @@ size_t Chunk::memory_usage(const MemoryUsageCalculationMode mode) const {
 std::vector<std::shared_ptr<const AbstractSegment>> Chunk::_get_segments_for_ids(
     const std::vector<ColumnID>& column_ids) const {
   DebugAssert(([&]() {
-                const auto column_count = static_cast<ColumnID>(column_count());
+                const auto number_of_columns = static_cast<ColumnID>(column_count());
```

Comment:
why?

Reply:
Because GCC did not compile otherwise.

Reply:
See here.

Reply:
Thanks, found it. For an easier lookup if anyone else is curious about it.
```
error: variable 'column_count' declared with deduced type 'const auto' cannot appear in its own initializer

                const auto column_count = static_cast<ColumnID>(column_count());
```

Reply:
Hmpf, this "see here" was supposed to include this link: https://hyrise-ci.epic-hpi.de/blue/organizations/jenkins/hyrise%2Fhyrise/detail/PR-2470/30/pipeline